build/tools/CFamilyJarPreProcessor/CFamilyJarPreProcessor.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Separate solution file for the tool. Not strictly required, but makes working in VS easier.